addon/components/form-field.js

```diff
@@ -62,12 +62,12 @@ export default Component.extend(formFieldOptions, {
   },
 
   actions: {
-    update(value) {
+    update(value, event) {
```

Comment:
Make it more general, so you'll pass everything the used field will pass to this update function after the value